### PR TITLE
Add icon to Banana Extension preference

### PR DESCRIPTION
**Pull request by codeimpl**
This patch adds icon to Banana Extension preference for user to
understand each preference.

Refs: #1086

### CODE REVIEW

**Comment by codeimpl** [comment]
I will update advanced_settings soon.

File: res/drawable/advanced_settings.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (jdragonbae) [inline res/drawable/advanced_settings.xml:1]: Please, add the license

Comment (codeimpl) [reply res/drawable/advanced_settings.xml:7]: done!

**Review by jdragonbae — APPROVED** [review]
lgtm

**Review by romandev — APPROVED** [review]
LGTM

**Pull request merged**